packages/dependency-extraction-webpack-plugin/README.md

```diff
@@ -135,6 +135,13 @@ Pass `useDefaults: false` to disable the default request handling.
 
 Force `wp-polyfill` to be included in each entry point's dependency list. This would be the same as adding `import '@wordpress/polyfill';` to each entry point.
 
+##### `externalizedReportFile`
+
+-   Type: string
```

Comment:
How about it’s both `string` and `boolean`? This way it’s `false` by default, it’s using some predefined path when `true` and a custom path otherwise.

Reply:
@gziolo I implemented it in https://github.com/WordPress/gutenberg/pull/35106/commits/81b2d9b185a14105049d27b25ea1e6ba3bf70876
and change names a little bit. Could you confirm the implementation and default name is fine to you?

Option name: `externalizedReport`
Option type: `boolean | string`
Default value: `false`
Default file name: `'externalized-dependencies.json'`